src/Microsoft.DotNet.Interactive.CSharpProject/CSharpProjectKernel.cs

```diff
@@ -277,23 +277,14 @@ private async Task<Workspace> AppendCodeToWorkspaceAsync(string code, int positi
     private static IEnumerable<Diagnostic> GetDiagnostics(string code, CompileResult result)
     {
         var diagnostics = Enumerable.Empty<SerializableDiagnostic>();
```

Comment:
This assignment could be moved to an `else` block below now.

Reply:
Good. I made the suggested change. Can you approve this cleanup change https://github.com/dotnet/interactive/pull/3574